components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.spec.ts

```diff
@@ -252,31 +264,6 @@ describe('ReportingComponent', () => {
       );
     });
 
-    it('parse end date', () => {
-      spyOn(reportQueryService, 'setState');
-      // end date of three days ago end_time=2019-09-05
-      const endDate = moment('2019-09-05', 'YYYY-MM-DD').utc().
-        startOf('day').add(12, 'hours');
-      const interval = 0;
-      const startDate = reportQueryService.findTimeIntervalStartDate(interval, endDate);
-
-      const reportQuery: ReportQuery = {
-        startDate: startDate,
-        endDate: endDate,
-        interval: interval,
-        filters: [
-          {type: {name: 'chef_tags'}, value: { text: '123' }}
-        ]
-      };
-
-      component.applyParamFilters([
-        {type: 'chef_tags', text: '123'},
-        {type: 'end_time', text: '2019-09-05'}]);
-      expect(reportQueryService.setState).toHaveBeenCalledWith(
```

Comment:
This test would not pass anymore. The 'getEndDate' test covers the same test case.